android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -404,8 +404,11 @@ public void call_sendMessage(String uuid, String content, String contentType, St
     final CallRecord callRecord = validateCallRecord(reactContext, UUID.fromString(uuid), promise);
     Objects.requireNonNull(callRecord);
 
-    final CallMessage callMessage = new CallMessage.Builder(validateMessageTypeFromString(messageType, promise))
-      .contentType(validateContentTypeFromString(contentType, promise)).content(content).build();
+    final CallMessage.MessageType _messageType = validateMessageTypeFromString(messageType, promise);
+    final String _contentType = validateContentTypeFromString(contentType, promise);
+
```

Comment:
Sorry, just one more thing. We're not actually modifying the control flow here yet. What happens with this control flow when `messageType` is invalid?

Reply:
added if statements to just return if validations are null